plugins/hypervisors/kvm/src/main/java/com/cloud/hypervisor/kvm/resource/IvsVifDriver.java

```diff
@@ -70,9 +71,9 @@ public void configure(Map<String, Object> params) throws ConfigurationException
         }
         _ivsIfUpPath = Script.findScript(utilScriptsDir, "qemu-ivs-ifup");
 
-        libvirtVersion = (Long) params.get("libvirtVersion");
```

Comment:
Looks like libvirtVersion was not being used on this class, is that right?

Reply:
Not being used indeed. Dead variable which was never called